lib/widgets/home_scaffold.dart

```diff
@@ -22,83 +25,101 @@ class _MyHomeScaffoldState extends State<HomeScaffold> {
-    ),
-    NavigationDestination(
-      icon: Icon(Icons.person),
-      label: 'Profile',
-    ),
-  ];
-
   final List<Widget> _pages = const <Widget>[
     HomePage(title: 'Home'),
     SearchPage(title: 'Search'),
     HomePage(title: 'Don\'t touch me :)'),
     LeaderboardPage(title: 'Leaderboard'),
     ProfilePage(title: 'Profile'),
   ];
-      
+
   @override
   Widget build(BuildContext context) {
-    return Center(
-      child: Scaffold(
-        appBar: AppBar(
-          title: const Text('Gomiko'),
-        ),
-        body: Center(
-          child: _pages.elementAt(_selectedIndex),
-        ),
-        floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
-        floatingActionButton: SizedBox(
-          height: 75,
-          width: 75,
-          child: FloatingActionButton(
-            //TODO: Add functionality to scan button
-            onPressed: () {},
-            tooltip: 'Scan',
-            shape: const CircleBorder(),
-            child: const Icon(
-              Icons.camera_alt,
-              size: 40,
-            ),
-          ),
-        ),
-        bottomNavigationBar: Stack(
-          alignment: Alignment.bottomCenter,
-          children: [
-            NavigationBar(
-              selectedIndex: _selectedIndex,
-              onDestinationSelected: _onDestinationSelected,
-              destinations: _destinations,
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Gomiko'),
+      ),
+      body: Center(
+        child: _pages.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: Transform.translate(
+        offset: const Offset(0, -24),
+        child: SafeArea(
+          child: Container(
+            padding:
+                const EdgeInsets.only(left: 12, top: 5, right: 12, bottom: 5),
+            margin: const EdgeInsets.symmetric(horizontal: 24),
+            decoration: BoxDecoration(
+              color: bottomNavBgColor.withOpacity(0.8),
+              borderRadius: BorderRadius.circular(24),
+              boxShadow: [
+                BoxShadow(
+                  color: bottomNavBgColor.withOpacity(0.3),
+                  offset: const Offset(0, 20),
+                  blurRadius: 20,
+                ),
+              ],
             ),
-            Positioned(
-              bottom: MediaQuery.of(context).size.height *
-                  0.03, // Adjust this to adjust the position of the text
-              child: const Text(
-                'Scan',
-              ),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                IconButton(
+                  onPressed: () {
+                    _onDestinationSelected(0);
+                  },
+                  icon: Icon(
+                    Iconsax.home,
+                    color: _selectedIndex == 0
+                        ? bottomNavSelectedColor
+                        : const Color(0xFF6E7191),
```

Comment:
Color is different than unselected color?

Reply:
forgot to update that value to use unselectedColor variable